project/plugins.sbt

```diff
@@ -15,3 +15,6 @@ addSbtPlugin("org.scalameta"                     % "sbt-mdoc"
 addSbtPlugin("org.scalameta"                     % "sbt-scalafmt"                  % "2.5.0")
 addSbtPlugin("pl.project13.scala"                % "sbt-jcstress"                  % "0.2.0")
 addSbtPlugin("pl.project13.scala"                % "sbt-jmh"                       % "0.4.3")
+addSbtPlugin("dev.zio"                           % "zio-sbt-website"               % "0.0.0+84-6fd7d64e-SNAPSHOT")
+
+resolvers += Resolver.sonatypeRepo("public")
```

Comment:
Isn't this redundant? I thought that it's already present by default.

Reply:
I remember, it cause not found issue when downloading snapshot artifacts. I can check it again.

Reply:
For snapshots you need
```scala
resolvers ++= Resolver.sonatypeOssRepos("snapshots")
```
(don't use `sonatypeRepo` anymore, always just `sonatypeOssRepos`)

Reply:
Thank you! I removed that line and it works properly for th `zio-prelude` project. I need to investigate more to find out why I need to explicitly set the resolver for snapshot versions in some repos.

Reply:
You need that :point_up: because you're using snapshot of `zio-sbt-website`

Reply:
In my local machine that works properly, maybe because of cached artifacts. Anyway, I added that line.